Toggl.iOS.Tests.UI/Configuration.cs

```diff
@@ -26,24 +22,16 @@ public static iOSApp GetApp()
             return app;
         }
 
-        private static void resetSimulator(string deviceId)
+        private static void reinstallApp(string deviceId)
         {
             if (deviceId == null)
             {
-                var preDeleteCmdLine = string.Format("simctl uninstall booted com.toggl.daneel.debug", deviceId);
+                var preDeleteCmdLine = string.Format("simctl uninstall booted com.toggl.daneel.debug");
```

Comment:
You don't seem to be using `Format` facilities, use normal string instead. It's not parameterized.

```suggestion
                var preDeleteCmdLine = "simctl uninstall booted com.toggl.daneel.debug";
```